powersimdata/design/generation/curtailment.py

```diff
@@ -0,0 +1,124 @@
+import pandas as pd
+
+from powersimdata.scenario.scenario import Scenario
+
+default_pmin_dict = {
+    "coal": None,
+    "dfo": 0,
+    "geothermal": 0.95,
+    "hydro": None,
+    "ng": 0,
+    "nuclear": 0.95,
+    "other": 0,
+    "solar": 0,
+    "wind": 0,
+    "wind_offshore": 0,
+}
```

Comment:
I believe 0.95 of `pmax` is standard for nuclear and geothermal. I would have imagined that `geothermal` would be dispatchable.

Reply:
I just encoded what we do right now in REISE.jl. Nuclear and geothermal are set to 95%, and everything else besides coal is set to 0.